### PR TITLE
add elfinder v2.1.12 with git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/Studio-42/elFinder
**Watch 185
  Star 1,981
 Fork 832**
@PeterDaveHello  could u help me check this PR for #5873 ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Missing `rm.wav`. 
https://github.com/maruilian11/cdnjs/blob/b9d69220b0c05d3f6a5947362d987764ab74a52d/ajax/libs/elfinder/2.1.12/js/elfinder.full.js#L2280

**Comment by maruilian11** [comment]
hmmm... i will add it
do we have a document about we host that type of file?

**Comment by PeterDaveHello** [comment]
There is no reason to mention everything, we host the files should be hosted.

**Comment by maruilian11** [comment]
@PeterDaveHello is new commit ok?

**Comment by PeterDaveHello** [comment]
Don't you need to update filemap of autoupdate?

**Comment by PeterDaveHello** [comment]
The only reason that your PR won't be merged is because of the quality and the wrong things, I don't know why you couldn't check it again though you've sent so many PRs.

**Comment by maruilian11** [comment]
I think maybe i always work on github when i am tired.
i will try to review the checklist when i resend a new commit.

File: ajax/libs/elfinder/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Studio-42/elFinder.git",
+    "basePath": "",
+    "files": [
+      "js/**/+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)",
+      "css/+(elfinder|theme)*.+(css|map)",
+      "img/*.+(png|gif)",
+      "sounds/rm.wav"
```
Comment (PeterDaveHello) [inline ajax/libs/elfinder/package.json]: What about `"sounds/*.wav"`?

**Comment by maruilian11** [comment]
@Amomo could you help me review this PR? thanks!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is ok.

**Pull request merged**